tests/xtro-sharpie/macOS-NetworkExtension.todo

```diff
@@ -13,11 +10,15 @@
 !missing-selector! NETunnelProviderManager::appRules not bound
 !missing-selector! NETunnelProviderManager::calendarDomains not bound
 !missing-selector! NETunnelProviderManager::contactsDomains not bound
-!missing-selector! NETunnelProviderManager::initForPerAppVPN not bound
 !missing-selector! NETunnelProviderManager::mailDomains not bound
 !missing-selector! NETunnelProviderManager::safariDomains not bound
 !missing-selector! NETunnelProviderManager::setAppRules: not bound
 !missing-selector! NETunnelProviderManager::setCalendarDomains: not bound
 !missing-selector! NETunnelProviderManager::setContactsDomains: not bound
 !missing-selector! NETunnelProviderManager::setMailDomains: not bound
 !missing-selector! NETunnelProviderManager::setSafariDomains: not bound
+!missing-enum! NEFilterReportFrequency not bound
+!missing-selector! +NETunnelProviderManager::forPerAppVPN not bound
```

Comment:
This looks weird - `+NETunnelProviderManager`?

Reply:
@whitneyschmidt why?

Reply:
weird because it starts with a `+` ?
That means it's a static selector

Reply:
🤦‍♀ @spouliot thank you for the kind reminder...